static/styles/compose.css

```diff
@@ -16,7 +16,7 @@
                 line-height: 1em;
             }
 
-            @media (min-width: 551px) {
+            @media (min-width: $xs-min) {
```

Comment:
For a change like this, can you comment on the commit message on whether you tested widths in the range in between and how it looked?  And feel free to post screenshots to support reasoning.

(I'd like to be sure we're at least noting places where we may be introducing a regression and opening issues)

Reply:
It might also help to mention the value of `xs` in the commit messages, so it's easy to look at how much the value is changing.

Reply:
(I don't need a proof that the changes are correct, but it's really helpful to know how things were tested so that I can reason about potential failure modes for this sort of refactoring.)

Reply:
Okay.